plugins/LanguagesManager/tests/Unit/TranslationWriter/Validate/CoreTranslationsTest.php

```diff
@@ -23,8 +23,7 @@ public function getFilterTestDataValid()
                     'General' => array_merge(array_fill(0, 251, 'test'), array(
                                                                               'Locale'          => 'de_DE.UTF-8',
                                                                               'TranslatorName'  => 'name'
-                                                                         )
-                    )
+                                                                         ))
```

Comment:
I think in instances like this we should add the trailing slash after the last element in the array and keep the formatting.. this type of change makes it less readable when it bunches brackets on one line and then ignores the indentation.

Reply:
I agree, but didn't want to spent time on improving this that will be changed with other rules anyway. Guess enabling the rules PSR2.Methods.FunctionCallSignature.Indent and PSR2.Methods.FunctionCallSignature.MultipleArguments will improve the readability here later.